### DISCUSSION TITLE
Markdown documentation requirement for self improvement.

### DISCUSSION THREAD

**Discussion by thatbeautifuldream** [thatbeautifuldream/java-dsa-bootcamp#17] [Q&A]
Requirement for contributors to contribute in the documentation to maintain this java bootcamp

**Comment by DEEZY4U** [comment]
Hey, @thatbeautifuldream I would like to contribute to the documentation.
By documentation, you mean the main ReadMe file of the repository right.

**Reply by thatbeautifuldream** [reply]
No I meant every subtopics in dsa overall.

**Reply by DEEZY4U** [reply]
Oh, should I make topic wise PR ?

**Reply by thatbeautifuldream** [reply]
I wanted this repository to have code as well as notes for the same topics divided topic wise with standard illustrations, I'll start pushing some files after i prepare in the `gh-pages` branch of the same repository.
Just so that there is cohesiveness in every subtopic of dsa and there is a one stop solution for interview preparation.

**Reply by DEEZY4U** [reply]
Ohh that's great , then I will also work on some note written on markdown and create some flowcharts.
